tests/sample/reflect_packet.c

```diff
@@ -100,6 +86,166 @@ reflect_packet(xdp_md_t* ctx)
         }
     }
 
+Done:
+    return rc;
+}
+
+int
+encapsulate_ipv4_reflect_packet(xdp_md_t* ctx)
+{
+    int rc = XDP_DROP;
+
+    // Adjust XDP context to allocate space for outer IPV4 header.
```

Comment:
```suggestion
    // Adjust XDP context to allocate space for outer IPv4 header.
```
for consistency with line 121